DailyReportWeb/Helpers/ReportRunner.cs

```diff
@@ -3,16 +3,22 @@
 using System.Linq;
 using System.Web;
 using Microsoft.Win32.TaskScheduler;
+using SourceControlLogReporter.Model;
+using System.Configuration;
+using SourceControlLogReporter;
 
 namespace DailyReportWeb.Helpers
 {
     public class ReportRunner
     {
-        public static bool TryRunReport(string id)
+        public static bool TryRunReport(string id, string draftKey="")
         {
             try
             {
-                RunReport(id);
+                if (string.IsNullOrEmpty(draftKey))
+                    RunReport(id);
+                else
+                    RunReportDirectly(id, draftKey, true);
```

Comment:
Nu mi se pare ca ai combinat in metoda asta atat RunReport cat si RunReportDirectly.
Si asta din cauza faptului ca numele a permis. Hai sa o redenumim TryRunReportTask si pe RunReport sa o redenumim in RunReportTask.

Iar verificarea asta cu draftKey trebuie facuta in controller.